### PR TITLE
feat: add Chinese Traditional translation

### PR DESCRIPTION
**Pull request by kane50613**
## Description

Added a new translation: Chinese (Traditional)

- [x] I have read and understand the [CONTRIBUTING.md](https://github.com/payloadcms/payload/blob/main/CONTRIBUTING.md) document in this repository.

## Type of change

- [ ] Chore (non-breaking change which does not add functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Change to the [templates](https://github.com/payloadcms/payload/tree/main/templates) directory (does not affect core functionality)
- [ ] Change to the [examples](https://github.com/payloadcms/payload/tree/main/examples) directory (does not affect core functionality)
- [ ] This change requires a documentation update

## Checklist:

- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Existing test suite passes locally with my changes
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/payload/src/translations/zh-tw.json [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    "selectLocales": "選擇要顯示的語言",
+    "selectVersionToCompare": "選擇要比較的版本",
+    "showLocales": "顯示語言：",
+    "showingVersionsFor": "顯示版本為：",
+    "status": "狀態",
+    "type": "類型",
+    "unpublish": "取消發佈",
+    "unpublishing": "取消發佈中...",
+    "version": "版本",
+    "versionCount_many": "發現{{count}}版本",
```
Comment (tyteen4a03) [inline packages/payload/src/translations/zh-tw.json:362]: Would be good to use `{{count}}個` for completeness

File: packages/payload/src/translations/zh-tw.json [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    "showAllLabel": "顯示所有{{label}}",
+    "sorryNotFound": "對不起，沒有找到您請求的東西。",
+    "sort": "排序",
+    "sortByLabelDirection": "按{{label}} {{direction}}排序",
+    "stayOnThisPage": "停留在此頁面",
+    "submissionSuccessful": "成功送出。",
+    "submit": "提交",
+    "successfullyCreated": "成功建立{{label}}",
+    "successfullyDuplicated": "成功複製{{label}}",
+    "thisLanguage": "Chinese (Traditional)",
```
Comment (DanRibbens) [inline packages/payload/src/translations/zh-tw.json:249]: This should be readable to somebody who only speaks this language. It is used in the dropdown selector for choosing your language. 

Examples:
ar: "العربية"
it: "Italiano"
vi: "Vietnamese (Tiếng Việt)" // even this is helpful

Comment (kane50613) [reply packages/payload/src/translations/zh-tw.json:249]: > This should be readable to somebody who only speaks this language. It is used in the dropdown selector for choosing your language.
> 
> Examples: ar: "العربية" it: "Italiano" vi: "Vietnamese (Tiếng Việt)" // even this is helpful

oh then the Chinese one should also be updated to 中文 (简体), i'll make another's commit later.

**Review by DanRibbens — APPROVED** [review]

**Comment by kane50613** [comment]
should I do something to pass that integration test

**Comment by DanRibbens** [comment]
We have a flaky postgres test. Thanks for offering, but this is completely unrelated to your PR.

**Pull request merged**